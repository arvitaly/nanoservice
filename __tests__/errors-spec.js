"use strict";
const errors_1 = require("./../errors");
describe("Errors", () => {
    it("unknown error type", () => {
        expect(errors_1.default.unknownTransportType("test1", "test2")).toMatchSnapshot();
    });
});
