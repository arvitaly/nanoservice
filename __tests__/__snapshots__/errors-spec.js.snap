exports[`Errors unknown error type 1`] = `"Unknown transport type:: test1 for transport \"test2\""`;
